Ukhu/Banka
# #165128574 Build POST /accounts endpoint

#### What does this PR do?
Test and build POST /accounts endpoint

#### Description of Task to be completed?
1. Write unit tests for the route
2. Set up the route
3. Add the necessary middlewares and handlers
3. Ensure test coverage is sufficient enough

#### How should this be manually tested?
1. Clone the repo
2. Run 'npm run test'
3. Start the server with the command 'npm run dev'
4. Send a post request on Postman to the signup route to obtain a token
5. Test on Postman using the following fields in the request body:
     'userId' - id of the newly created user
     'type' - type of account to be created (can either be 'current' or 'savings')
     'token' - jwt token gotten from the response from the signup endpoint
6. The response gotten should be a JSON object with a status property and a data or error(s) property for success and failure respectively.

#### Any background context you want to provide?
N/A

#### What are the relevant pivotal tracker stories?
#165128574

#### Screenshots (if appropriate)
N/A

#### Questions:
N/A
